### DISCUSSION TITLE
TensorFlow Multi-GPU?

### DISCUSSION THREAD

**Discussion by TortoiseHam** [securefederatedai/openfederatedlearning#122] [Q&A] [answered]
Has the repo been validated in a multi-GPU environment yet? I'm having some problems there but not sure whether to raise an Issue if it's not a supported use case

**Comment by alexey-gruzdev** [comment, marked as answer]
Hi @TortoiseHam. We didn't validate a multi-GPU case for TF/PyTorch.
